addons/l10n_jo/data/l10n_jo_chart_data.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+     <record id="jordan_chart_template_standard" model="account.chart.template">
+         <field name="name">Jordan Chart of Accounts - Standard</field>
+         <field name="code_digits">6</field>
+         <field name="bank_account_code_prefix">1001</field> <!-- TODO check what this is -->
+         <field name="cash_account_code_prefix">1002</field>
+         <field name="transfer_account_code_prefix">1001</field>
```

Comment:
It does not make any sense that this is the same as the bank one.   (this should be the intermediary account)